.setup/scripts/utils.sh

```
@@ -498,30 +497,33 @@ function post_setup_12 {
-  sed -i -e "s/base: ht\//base: \//g" /var/www/html/.Build/$VERSION/config/sites/main/config.yaml
-  sed -i -e 's/base: \/en\//base: \//g' /var/www/html/.Build/$VERSION/config/sites/main/config.yaml
+  sed -i -E "s|^base: .*|base: /|" /var/www/html/.Build/$VERSION/config/sites/main/config.yaml
 }
 
 # Function to perform post-setup tasks for TYPO3 version 13.
 # It creates the TYPO3 database, sets up TYPO3 by running the installation setup,
 # configures TYPO3 settings, and modifies configuration files to enable deprecations.
 function post_setup_13 {
   mysql -h db -u root -p"root" -e "CREATE DATABASE $DATABASE;"
-  $TYPO3_BIN  setup -n --dbname=$DATABASE --password=$TYPO3_DB_PASSWORD --create-site="https://${VERSION}.${EXTENSION_NAME}.ddev.site" --admin-user-password=$TYPO3_SETUP_ADMIN_PASSWORD
+  $TYPO3_BIN  setup -n --dbname=$DATABASE --password=$TYPO3_DB_PASSWORD --create-site="https://${VERSION}.${DDEV_SITENAME}.${DDEV_TLD}" --admin-user-password=$TYPO3_SETUP_ADMIN_PASSWORD
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)